src/ripple/app/tx/tests/DeliverMin.test.cpp

```diff
@@ -0,0 +1,70 @@
+//------------------------------------------------------------------------------
+/*
+  This file is part of rippled: https://github.com/ripple/rippled
+  Copyright (c) 2012-2015 Ripple Labs Inc.
+
+  Permission to use, copy, modify, and/or distribute this software for any
+  purpose  with  or without fee is hereby granted, provided that the above
+  copyright notice and this permission notice appear in all copies.
+
+  THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+  WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+  MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+  ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+  WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+  ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <BeastConfig.h>
+#include <ripple/test/jtx.h>
+#include <beast/unit_test/suite.h>
+
+namespace ripple {
+namespace test {
+
+using namespace jtx;
+
+class DeliverMin_test : public beast::unit_test::suite
+{
+public:
+    void
+    test_payment_deliver_min()
+    {
+        testcase("payment deliver minimum");
+
+        Env env(*this);
+        Account const alice("alice");
+        auto const USD = alice["USD"];
+        Account const bob("bob");
+
+        env.fund(XRP(10000), alice, bob);
+        env.require(balance(alice, XRP(10000)));
+        env.require(balance(bob, XRP(10000)));
```

Comment:
These `require` statements are not needed. First of all, we're not testing the `Env::fund` function and second, `Env::fund` already performs such consistency checks internally:
https://github.com/ripple/rippled/blob/develop/src/ripple/test/jtx/impl/Env.cpp#L171